src/shared_utilities/drivers.jl

```diff
@@ -358,18 +358,27 @@ function turbulent_fluxes_at_a_point(
     _LH_v0::FT = LP.LH_v0(earth_param_set)
     _ρ_liq::FT = LP.ρ_cloud_liq(earth_param_set)
 
+    # aerodynamic resistance
+    r_ae::FT = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc))
+
+    # latent heat flux
+    E0::FT = SurfaceFluxes.evaporation(surface_flux_params, sc, conditions.Ch)
+    E = E0 * r_ae / (r_sfc + r_ae)
+    LH = _LH_v0 * E
+
+    # sensible heat flux
     cp_m::FT = Thermodynamics.cp_m(thermo_params, ts_in)
     T_in::FT = Thermodynamics.air_temperature(thermo_params, ts_in)
     ΔT = T_in - T_sfc
-    r_ae::FT = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc))
     ρ_air::FT = Thermodynamics.air_density(thermo_params, ts_in)
+    grav = SFP.grav(surface_flux_params)
+    ΔΦ = grav * Δz(sc)
```

Comment:
I rearranged the code here so the diff is larger than it needs to be, but the only physical change is adding this `\Delta \Phi` term to the SHF calculation